src/Analyzers/Core/CodeFixes/Formatting/FormattingCodeFixProvider.cs

```diff
@@ -23,16 +23,38 @@ public sealed override ImmutableArray<string> FixableDiagnosticIds
 
         protected abstract ISyntaxFormatting SyntaxFormatting { get; }
 
+#if !CODE_STYLE
+
+        /// <summary>
+        /// Fixing formatting is high priority.  It's something the user wants to be able to fix quickly, is driven by
+        /// them acting on an error reported in code, and can be computed fast as it only uses syntax not semantics.
+        /// It's also the 8th most common fix that people use, and is picked almost all the times it is shown.
+        /// </summary>
+        private protected override CodeActionRequestPriority ComputeRequestPriority()
+            => CodeActionRequestPriority.High;
```

Comment:
I believe you also need to update the Priority for the corresponding analyzer. There is a property on IBuiltInAnalyzer for the same.